com.unity.ml-agents/Runtime/Analytics/InferenceAnalytics.cs

```diff
@@ -156,7 +156,7 @@ IList<IActuator> actuators
             var inferenceEvent = new InferenceEvent();
 
             // Hash the behavior name so that there's no concern about PII or "secret" data being leaked.
-            inferenceEvent.BehaviorName = AnalyticsUtils.Hash(behaviorName);
+            inferenceEvent.BehaviorName = AnalyticsUtils.Hash(k_VendorKey, behaviorName);
```

Comment:
There are other hashing in this file too ( eg. L222) should we change them to hmac as well?

Reply:
In this instance it looks like it's being used just for object identification. I don't see an argument that the model's layer data is itself PII and so the weaker hash is likely fine here.